cypress/e2e/gui/paginaInicial.cy.js

```diff
@@ -0,0 +1,19 @@
+/// <reference types="cypress" />
+
+describe('Página inicial - Saudação', () => {
+    it('Saudação Padrão', () => {
+        cy.visit('http://localhost:3000/')
+
+        cy.get('[data-testid="table"] > :nth-child(1)')
```

Comment:
Aqui está outro seletor não-descritivo.

Que tal algo como o seguinte: `cy.contains('p', 'Hi there!').should('be.visible')`

Reply:
obrigada pela dica